front/src/applications/rollingStockEditor/consts.ts

```diff
@@ -56,25 +71,14 @@ export enum Metadata {
 export enum Parameter {
```

Comment:
Same here, can you rename it more precisely and transform it into an array ?

Reply:
done